performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -162,6 +162,9 @@ class AnalysisResult
 #pragma db transient
 #endif
   rusage m_sys_usage;
+#ifdef PERFORMANCE_TEST_ODB_FOR_SQL_ENABLED
+#pragma db transient
```

Comment:
`#pragma db transient` is used to specify that the variable below (in this case `m_cpu_info`) should not be saved to the database, see https://www.codesynthesis.com/products/odb/doc/odb-manual.pdf 14.4.11. The warnings are also expected, you can read about this in section 14.9 in the manual.

Since in our existing database `m_cpu_info` is not stored and the schema migration branch is not merged yet (https://github.com/ApexAI/performance_test/pull/112) we have to have this member as transient, otherwise the program will crash.